src/rtpghi~.c

```diff
@@ -169,6 +267,7 @@ void *rtpghi_tilde_new(t_symbol *s, int argc, t_atom *argv)
 	post("win_def %p, do_causal %f, tol %f, ol %f", x->window_type_pd, x->do_causal_pd, x->tol_pd, x->ol_pd);
     x->sta_pd=NULL;
     x->c = NULL;
+	x->blocksize = 0;
```

Comment:
typically there's no need to set the members to `0`, as `pd_new()` already does this (but of course, there's no harm in doing so)